tools/gcpviz/gcpviz.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+if [ ! -d /gcpviz/cai ] ; then
+    echo "Missing directory /gcpviz/cai!"
+    exit 1
+fi
+
+cd /gcpviz
```

Comment:
Please use 'cd /gcpviz || exit' in case cd fails

Reply:
Added,.